satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -390,3 +390,25 @@ read one (
 	select certRecord
 	where  certRecord.id = ?
 )
+
+//--- bwagreement_bucket ---//
+
+model bwagreement_bucket (
+	key id
+
+	field id              blob
+	field serialnum       bwagreement.serialnum cascade
```

Comment:
This would need to be `bytes`, because of other reasons. With hammer it's going to be a bytes, it was previously encoded as a string, for unfortunate reasons. I'm not sure what the effects of cascade deletes will be with large tables.

Reply:
I haven't made any changes to this yet.  We need to test the performance of this first.  I'm going to work with stefan to test it. Will post results once we have them.

Reply:
ah, also... use `serial_number`, we need to fix the other eventually as well